### DISCUSSION TITLE
Accuracy and Benchmarks

### DISCUSSION THREAD

**Discussion by oak-innovation** [deepgram/community#625] [General help]
Deepgram is mentioning benchmark showing that their Nova-2 model is faster and more accurate than for instance OpenAI's Whisper. However When I try to translate Swiss German it turns out that Whisper is more accurate. Especially if there are e.g. geographical names. 

My impression is, that Whisper has a downstram NLP enginge that tries to correct what whas misspelled by the sound engine.  

Can you tell me something about German (and Swissgerman), and about the influence of the sound quality? Are Deepgrams benachmarks only for English? What can I change in order to unleash Nova-2's power?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
